internal/worker/server.go

```diff
@@ -551,16 +546,63 @@ func (s *Server) FinishJob(token uuid.UUID, result json.RawMessage) error {
 		}
 	}
 
-	var jobResult JobResult
-	jobInfo, err := s.jobStatus(jobId, &jobResult)
+	jobType, err := s.JobType(jobId)
 	if err != nil {
-		logrus.Errorf("error finding job status: %v", err)
-	} else {
-		statusCode := clienterrors.GetStatusCode(jobResult.JobError)
-		arch := getBuildJobArch(jobInfo.JobType, result)
-		prometheus.FinishJobMetrics(jobInfo.JobStatus.Started, jobInfo.JobStatus.Finished, jobInfo.JobStatus.Canceled, jobInfo.JobType, jobInfo.Channel, arch, statusCode)
+		return err
+	}
+
+	var arch = ""
```

Comment:
nitpick: you should have defined the variable by specifying type `var arch string`, instead of assigning a value, similar to the variable on the line below this one. The variable would be initialized to the default value for `string`, which is an empty string `""`.

Reply:
Looks like there is something wrong with the RPMS so if I have to rebase the PR then I will fix this nitpick